test/bdd/pkg/vc/vc_steps.go

```diff
@@ -320,7 +309,7 @@ func (e *Steps) createProfileAndPresentation(user, credential, did, privateKey,
 
 	ldpContext := &verifiable.LinkedDataProofContext{
 		SignatureType:           "Ed25519Signature2018",
-		SignatureRepresentation: getSignatureRepresentation("JWS"),
+		SignatureRepresentation: bddutil.GetSignatureRepresentation("JWS"),
```

Comment:
Do we have tests for both `Ed25519Signature2018` and `JsonWebSignature2020`?

Reply:
Yes - https://github.com/trustbloc/edge-service/pull/342/files#diff-b65ece61ea44b974d209834e4299072cR19-R20